databend-client/src/main/java/com/databend/client/ParseJsonDataUtils.java

```diff
@@ -33,7 +33,7 @@ private ParseJsonDataUtils() {}
      * input List<List<Object>> : a list of rows parsed from QueryResponse
      * output Iterable<List<Object>> : convert the input rows into DatabendType and return an immutable list
      */
-    public static Iterable<List<Object>> parseRawData(List<QueryRowField> schema, List<List<Object>> data)
+    public static List<List<Object>> parseRawData(List<QueryRowField> schema, List<List<Object>> data)
```

Comment:
No need to use `List` here, `Iterable` is enough

Reply:
need to be able to check for empty.